action.yml

```diff
@@ -59,6 +59,9 @@ inputs:
   job-status:
     description: 'Workaround to pass job status to post job step. This variable is not intended for manual setting'
     default: ${{ job.status }}
+  token:
+    description: Used to pull java versions from setup-java.  Since there's a default, this is typically not supplied by the user.
```

Comment:
```suggestion
    description: `Used to pull java versions from setup-java. Since there's a default, this is typically not supplied by the user`
```